public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -235,9 +235,12 @@ export class PanelInspector extends PureComponent<Props, State> {
               if (width === 0) {
                 return null;
               }
+              const minWidth = 150 * processed[selected].fields.length;
```

Comment:
Wonder if this logic should be in the table component instead of every place we use the table component

Reply:
and provide minColumnWidth as a prop?

Reply:
Yea something like that, and the table component also needs to take I to account manual set column width on each field, so there is more space available for other columns (think this is done already)

Reply:
Yes I _pretty sure_ field overrides still work. Will do some testing